framework-v1/README.md

```diff
@@ -147,10 +147,27 @@ Election results will be announced on the NDC’s Medium, Telegram, and Discord
 
 After the election results are announced, the elected representatives will convene for the first time in the first NDC governance town hall, where we will hear from elected members from the three branches of NDC governance.
 
-## More links
-
-1. [fair-voting-policy](https://bafkreiapsavs6hrc6aomzagub7aumogkfbjewlo2tvwzesr4myshlecfqy.ipfs.nftstorage.link)
-2. [Registry docs](https://github.com/near-ndc/i-am-human/tree/master/contracts/registry#readme)
-3. [Elections smart contract](https://github.com/near-ndc/voting-v1/tree/master/elections)
-4. [I Am Human (IAH)](https://i-am-human.gitbook.io/i-am-human-docs/)
-5. [Whistleblower program](https://medium.com/@neardigitalcollective/introducing-ndc-whistleblower-bounty-program-d4fe1b9fc5a0)
+## References
+
+1. [NDC Overview]( https://docs.google.com/presentation/d/1Vm12P-vMtaDHvK-gtFroW6PAQ6fis32SiGHbW2qYaGQ/edit?usp=sharing)
```

Comment:
please use the updated link:
https://docs.google.com/presentation/d/1YFzsBRB-UcZYN93tDVEHUGE7uGx2OuW61PnXsNMYgPU/edit?usp=sharing

Reply:
OK, updated